pipelines/README.md

```diff
@@ -182,6 +182,7 @@ Be sure to also use the correct config map with the test data.
 # From the root folder where this README is located
 oc create -f tekton/aiedge-e2e/aiedge-e2e.pipelinerun.yaml
 ```
+Note: It is recommended to use VolumeClaimTemplate if the usecase is to run multiple pipelineruns concurrently, otherwise just make sure you don't run two of the same model concurrently.
```

Comment:
```suggestion
> [!IMPORTANT]
> Since the `buildah-cache` workspace is used to share data between TaskRuns in a PipelineRun, a PersistentVolumeClaim type VolumeSource is required to fulfill it properly.
> We strongly recommend that this is fulfilled using the `volumeClaimTemplate` approach, rather than the `persistentVolumeClaim` approach.
> If you must use the `persistentVolumeClaim` approach to re-use an existing PersistentVolumeClaim, then you will likely hit issues if two PipelineRuns for the same model name are executed concurrently (and possibly other corner cases).
> See the Tekton documentation around [Using PersistentVolumeClaims as VolumeSource](https://tekton.dev/docs/pipelines/workspaces/#using-persistentvolumeclaims-as-volumesource).
```

See https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts